recipes/clinod/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eu -o pipefail
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cp -R * $outdir/
+cp $RECIPE_DIR/clinod.py $outdir/clinod
+ls -l $outdir
+ln -s $outdir/clinod $PREFIX/bin
+chmod 0755 "${PREFIX}/bin/clinod"
+
+# clinod expects the batchman binary from SNNS in its folder:
+ln -s "`which batchman`" $outdir/batchman
```

Comment:
Yes to the first. The second will not work as you don't have $outdir defined in the runtime stage.
Eseentially, you have 3 stages. 
* Build stage, all in build.sh
* test/run-time stage your tests in meta.yaml - conda is still here
* And the BioConda specific one: The mulled stage: No conda no nothing just your package in a busybox.

All 3 are isolated.